ref: Remove next.js EA alert (#3764)
getsentry/sentry-docs: src/includes/getting-started-primer/javascript.nextjs.mdx

- Automatic [Performance Monitoring](/product/performance/) for both the client and server, from version `6.5.0`

Under the hood the SDK relies on our [React SDK](/platforms/javascript/guides/react/) on the frontend and [Node SDK](/platforms/node) on the backend, which makes all features available in those SDKs also available in this SDK.

<Alert level="info" title="Early Adopter">

> When using the SDK on Vercel production environment, there currently is a known issue with Sentry not being able to pick up errors/transaction when calling serverless API routes. We are working on resolving this.

Reviewer: I'd recommend that we add a link to the GH issue where we're tracking this for "[this issue]" a the end of the second sentence.
```suggestion
In Vercel production environments, the SDK is currently unable to pick up errors and/or transactions when calling serverless API routes. We are working to resolve this issue.
```
